docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -1,240 +1,396 @@
+\\
+\textbf{Milestone Coordinator}: The milestone coordinator has the responsibility of facilitating the creation and completion of deliverables. They are also responsible for assuring that each deliverable meets the criteria provided and that the standard of excellence is being maintained throughout the duration of the project.
+\\
+\textbf{Documentation Coordinator}: The documentation coordinator reviews documentation as the project progresses, ensuring that documents have a consistent level of quality and reflects the current state of the project. They work with team members to update, organize, and maintain a variety of documents, while also implementing best practices and managing version control.
+\\
+\textbf{QA Tester}: The QA tester is responsible for evaluating software to identify bugs and ensure it meets quality standards. They design and execute test plans and collaborate with the development team to address issues, ensuring a high-quality user experience.
+\\
+\textbf{Systems Architect}: The systems architect is responsible for designing the overall architecture of a software system, ensuring that it meets both technical and business requirements.
+\\
+\textbf{Frontend Specialist}: The frontend specialist focuses on creating the user interface and user experience of a web application, using technologies like HTML, CSS, and JavaScript. They work to ensure that the application is visually appealing, responsive, and user-friendly, collaborating closely with designers and backend developers.
+\\
+\textbf{Backend Specialist}: The backend specialist is responsible for server-side logic, database interactions, and application functionality. They build and maintain the core application infrastructure using programming languages and frameworks, ensuring that the frontend and backend communicate seamlessly to deliver a smooth and efficient user experience.
+\\
+\textbf{User Researcher}: The user researcher conducts studies to understand user behaviors, needs, and motivations through methods like interviews, surveys, and usability testing. They analyze the findings to provide insights that enhance product design and development, ensuring that the final product aligns with user expectations
 
 \section{Workflow Plan}
 
 \begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
-	classification, etc.?
-  \item Use of CI/CD
+    \item How will you be using git, including branches, pull request, etc.?
+    \item How will you be managing issues, including template issues, issue classification, etc.?
+    \item Use of CI/CD
 \end{itemize}
 
-\section{Project Decomposition and Scheduling}
-
+\subsection{GitHub Issues}
 \begin{itemize}
-  \item How will you be using GitHub projects?
-  \item Include a link to your GitHub project
+    \item will be used to track and manage all development tasks.
+    \begin{itemize}
```

Comment:
not sure if we wanna specify how the work will be divided up? (i.e. horizontal slicing vs. vertical)